t/02attean.t

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use Attean;
+use Test::Requires { 'Attean::IRI' => '0.024',
```

Comment:
This isn't complete. Types::Attean and Types::Namespace are required for this test.

Reply:
OK, how about now?

I'm a bit surprised I need to say `use Attean` before the `Test::Requires`, but it skips without that.